src/ServiceInsight/ServiceControl/DefaultServiceControl.cs

```diff
@@ -537,6 +564,7 @@ void LogError(IRestResponse response)
         static bool HasSucceeded(IRestResponse response) => successCodes.Any(x => response != null && x == response.StatusCode && response.ErrorException == null);
 
         static IEnumerable<HttpStatusCode> successCodes = new[] { HttpStatusCode.OK, HttpStatusCode.Accepted, HttpStatusCode.NotModified, HttpStatusCode.NoContent };
+        static Regex linkExpression = new Regex(@"<([^>]+)>;\s?rel=""(\w+)"",?\s?", RegexOptions.Compiled);
```

Comment:
Isn't string operations simple enough for this purpose? Whenever I see a regex I cringe.